dependencies/prepare.sh

```diff
@@ -11,7 +11,7 @@ DEBIAN_ARCH=$TARGETARCH
 if [ "$TARGETARCH" = "arm" ]
 then
   DEBIAN_ARCH=armhf
-  if [ "$TARGETVARIANT" = "v5" ]
+  if [ "$TARGETVARIANT" = "v5" || "$TARGETVARIANT" = "v6" ]
```

Comment:
As both arch's are dropped, don't we need to remove this `if`?

Reply:
I hope to cover all archs here since our code can be compiled with other arm archs.

Updated to emphasize `v7`.

Reply:
Hmm, I thought that we drop the support of `v6` completely from our product. Would you like to keep it for the direct host deployment scenario?
I'm concerned about the possible complexity of our code and scripts in the case of different lists of arch's for different deployments.

Reply:
I want to follow the original script and cover other archs in the script, at least covering what Debian supports. We drop `arm/v6` because of dependencies. The code still works with `arm/v6` aka `armel`. [The original script](https://github.com/photoview/photoview/blob/a6cbfc76f3d4acbf9dd3e5767a34ba319fc4cd56/scripts/set_compiler_env.sh#L56) supports `ppc` and `s390x` even we don't say we support them.

Reply:
But you've created this original script, as far as I remember, right?
Ok, as there is no negative impact of having these arch's there, I'm ok if you'd like to keep them. Just think that this is a 'dead code'. We could get back to it later when doing some other changes in that place.

Reply:
It renamed from [`go_wrapper.sh`](https://github.com/photoview/photoview/blob/6693920b61b1afda29786566b418ba7df9ffd628/docker/go_wrapper.sh).

If we drop other archs in the script, I'd like to fail the script with non-supported archs. How about that?

Reply:
Ah, I see.
Let's keep it as is for now, as there is no negative impact. We always could remove those archs later when we are faced with an issue